docs/artifactory/npminstall/help.go

```diff
@@ -2,7 +2,4 @@ package npminstall
 
 const Description = "Run npm install."
 
-var Usage = []string{`jfrog rt npmi [command options] <repository name>`}
-
-const Arguments string = `	repository name
-		The source npm repository. Can be a local, remote or virtual npm repository.`
+var Usage = []string{`jfrog rt npmi [command options]`}
```

Comment:
Missing *npm args* (see the usage of the go and nuget commands).

Reply:
Fixed.